22_pipeslab/pipeslab.txt

```diff
@@ -30,5 +32,9 @@ $ who | cut -d ' ' -f 3 | sort | uniq -c
 What was printed? Why? Fix the command line to get the output:
 2 Dec
 2 Nov
+Only the result of the uniq number from the count operation.
+It was because of the parameter "f" that was sent to the sort and uniq
+who | cut -d ' ' -f 17 | sort | uniq -c
+
```

Comment:
Partially works. `who` has different output style on each machine and the number 17 which works for you may not be correct for someone else.

The reason cut is wrong is its unable to work well with duplicate spaces. A better solution would be to remove the duplicate spaces before the command and use:

` who | tr -s ' ' | cut -d ' ' -f 3 | sort | uniq -c`